docs/Example HOWTO.md

```diff
@@ -320,3 +383,150 @@ dns2.example.com.    IN      A       192.168.0.20
+            2419200     ; Expire
+             604800 )   ; Negative Cache TTL
+
+;Then we define our DNS servers for this zone. End-points should be configured with one or both of these DNS server
+
+; DNS servers
+        IN      NS      dns1.example.com.
+        IN      NS      dns2.example.com.
+
+;We add the following PTR records to indicate the server supports Service Discovery.  Clients looking for DNS-SD support will query the server for these specific records (`b` is for `browse`,  `lb` is for `legacy browse`.)
+
+; These lines indicate to clients that this server supports DNS Service Discovery
+b._dns-sd._udp  IN      PTR     @
+lb._dns-sd._udp IN      PTR     @
+
+;Next we define the NMOS services provided by this server:
+
+; These lines indicate to clients which NMOS service types this server advertises:
+_services._dns-sd._udp  PTR     _nmos-register._tcp
+_services._dns-sd._udp  PTR     _nmos-query._tcp
+
+;There should be one `PTR` record for each instance of the service you wish to advertise. 
+; Here we have  service available through the Registration API and one service available through the Query API:
+
+_nmos-register._tcp     PTR     reg-api-1._nmos-register._tcp
+_nmos-register._tcp     PTR     reg-api-2._nmos-register._tcp
+_nmos-query._tcp        PTR     qry-api-1._nmos-query._tcp
+
+
+;Now we add `SRV` records that return the URL for the registration and query servers.  In this case, both of the records point to `rds1.example.com`.
+
+; NMOS RDS and Query Services       Type   Priority  Weight   Port   Host 
+;reg-api-1._nmos-register._tcp        SRV     10        10      80    rds1.example.com.
+;reg-api-2._nmos-register._tcp        SRV     20        10      80    rds2.example.com.
+
+; NMOS RDS services
+; High Priority RDS                             TTL   Class  SRV  Priority  Weight  Port     Target
+reg-api-1._nmos-register._tcp.example.com.     3600    IN    SRV     10       10      80      rds1.example.com.
+
+; Lower Priority RDS
+reg-api-2._nmos-register._tcp.example.com.     3600    IN    SRV     20       10      80      rds2.example.com.
+
+qry-api-1._nmos-query._tcp           SRV     10        10      80    rds1.example.com.
+
+;We add `TXT` records which provide information relevant to the IS-04 specification
+
+; Additional metadata relevant to the IS-04 specification. See IS-04 specification section "Discovery: Registered Operation"
+reg-api-1._nmos-register._tcp        TXT     "api_ver=v1.0,v1.1,v1.2,v1.3" "api_proto=http" "pri=10" "api_auth=false"
+reg-api-2._nmos-register._tcp        TXT     "api_ver=v1.0,v1.1,v1.2,v1.3" "api_proto=https" "pri=20" "api_auth=false"
+
+qry-api-1._nmos-query._tcp           TXT     "api_ver=v1.0,v1.1,v1.2,v1.3" "api_proto=http" "pri=0" "api_auth=false"
+
+;In both cases above the `SRV` records tell clients to access the server using port `80`. This would suit default HTTP access, but if HTTPS is used, this would need to be changed to `443`.
```

Comment:
```suggestion
;In both cases above the `SRV` records tell clients to access the server using port `80`. This would suit default HTTP access (indicated by `api_proto=http` in the `TXT` records). If HTTPS is used, this would need to be changed to `443`.
```